docker-compose.yaml

```diff
@@ -0,0 +1,77 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '3.2'
+services:
+  # PostgreSQL database.
+  postgres:
+    image: 'launcher.gcr.io/google/postgresql9'
+    volumes:
+    - './scripts:/docker-entrypoint-initdb.d'
+    environment:
+    - 'POSTGRES_PASSWORD=postgres'
```

Comment:
I don't have enough context to ask a refined question, but would there be any reason to hide fields associated with password?

Reply:
In this case the database is not exposed to the outside world (no "ports" field [1]) so I believe the plaintext password is acceptable. I'll add a comment to this effect.

[1] https://docs.docker.com/compose/compose-file/#ports